src/credentials-manager/la-policies.js

```diff
@@ -0,0 +1,6 @@
+const SecurityLevel = {
```

Comment:
Not sure security level is the appropriate term for this, as it implies that one option is necessarily more secure than the other.

Reply:
In any case, this file also needs to be renamed.

Reply:
Changed it back to LAPolicy :\ 

The reason I changed the name in the first place was 
1. I wanted to use something common so that we can use it if Android introduces this feature as well
2. Expo uses the name `SecurityLevel` for this feature

But I am not super convinced with the naming so I am going back to LAPolicy.